storage/common_flags.go

```diff
@@ -21,6 +21,7 @@ import (
 
 var ArgDbUsername = flag.String("storage_driver_user", "root", "database username")
 var ArgDbPassword = flag.String("storage_driver_password", "root", "database password")
+var ArgDbPasswordFile = flag.String("storage_driver_password_file", "", "database password from file")
```

Comment:
can you add that it takes precedence over --storage_driver_password?

Reply:
note added.